pages/styles/teams.css

```diff
@@ -0,0 +1,24 @@
+.boardMembers {
+  align-items: flex-start;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+}
+
+.staffMembers {
+  padding: 20px 0;
+}
+
+.foundingMembers {
+  padding: 2rem 0 0 0;
+}
+
+@media screen and (max-width: 840px) {
```

Comment:
let's use one of our viewport dimensions here in a custom media query!

Check out this PR for examples: https://github.com/OperationCode/front-end/pull/333/files